pkg/command/context.go

```diff
@@ -1517,6 +1520,25 @@ func printContext(writer io.Writer, ctx *configtypes.Context) {
 	outputWriter.Render()
 }
 
+// getTanzuResourceScopePath returns the scope path based on the ResourceInfo
+// TODO: Update this function when CLI context provides support for cluster/CF resources(CF Foundation/CF Org/CF Space)
+func getTanzuResourceScopePath(resources *config.ResourceInfo) string {
+	scopePath := ""
+	if resources.OrgID != "" {
+		scopePath += "/"
+	}
```

Comment:
nit: (because these are corner cases and unlikely to happen unless the context is corrupted)

not sure if you intended the fallthrough when orgid or projectname is somehow empty, because the result would be an "invalid" path that does not follow scope hierarchy rules.

Reply:
Good point. Thanks for pointing out. Updated the PR to address the cases where orgid / projectname is empty.